OnboardingMarsV4/Pages/Language.cs

```diff
@@ -0,0 +1,246 @@
+﻿using MarsOnboardV2.Drivers;
+using MarsOnboardV2.Support;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.DevTools.V121.FedCm;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace MarsOnboardV2.Pages
+{
+    public class Language : CommonDriver
+    {
+        public IWebElement element;
+        public string xpathlasttable;
+        public string xpathduplicate1;
+        public string xpathduplicate2;
+        public string xpathupdate;
+
+        public Language()
+        {
+            element = null;
+            xpathlasttable = "//div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[1]";
+            xpathduplicate1 = "//div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[";
+            xpathduplicate2 = "]/tr/td[1]";
+            xpathupdate = "//div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[1]/tr/td[1]";
+
+        }
+
+        public IWebElement GetElement(string path)
+        {
+            element = driver.FindElement(By.XPath(path));
```

Comment:
So your find by can be moved up to the instance level check this example https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

Have a look at the calculator page object class

Reply:
Thanks, I will work on it.